src/main.rs

```diff
@@ -263,11 +285,26 @@ fn main() {
         .version("0.1.0")
         .author("Leo Gaspard <todiff@leo.gaspard.ninja>")
         .about("Diffs two todo.txt files")
-        .args_from_usage(
-            "<BEFORE>     'The file to diff from'
-            <AFTER>        'The file to diff to'")
+        .args_from_usage("
+            <BEFORE>        'The file to diff from'
+            <AFTER>         'The file to diff to'
+        ")
+        .arg(clap::Arg::with_name("color")
+            .long("color")
+            .takes_value(true)
+            .possible_values(&["auto", "always", "never"])
+            .default_value("auto")
+            .help("Colorize the output"))
         .get_matches();
 
+    let color_option = matches.value_of("color").expect("Internal error E001.1");
```

Comment:
Should be called `E009`, with the current numbering (there's no logic in the way internal error numbers are allocated here, just trying to give a unique code to each internal error, and having Exxx looks like an acceptable numerotation)